Dockerfile

```diff
@@ -60,5 +68,6 @@ RUN python3.8 -m pip install pip==${PIP_VERSION} \
   && pip install --no-cache "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"
 
 COPY unstructured unstructured
+COPY test_unstructured test_unstructured
```

Comment:
we probably don't want to include these in the docker image right? maybe just a copy step before running the tests on the container in CI?

Reply:
I'd suggest mounting the test dir in during `make docker-test`. To ryan's point on the pip install, we could even get fancy and mount the local .venv to bring the test deps in at runtime. But yeah, certainly not a priority.